drivers/src/sha2_512_384acc.rs

```diff
@@ -214,6 +212,58 @@ impl Sha2_512_384AccOp<'_> {
         Ok(())
     }
 
+    /// Prepare the SHA accelerator for streaming.
+    ///
+    /// # Arguments
+    ///
+    /// * `dlen` - length of data that will be hashed
+    /// * `maintain_data_endianess` - reorder byte endianess if false, leave as-is if true
+    pub fn stream_start_512(
+        &mut self,
+        dlen: u32,
+        maintain_data_endianess: bool,
```

Comment:
Super nit: Using an enum may make the calling code easier to read:

```rust
sha.stream_start_512(64, StreamEndianness::Native)?;
sha.stream_start_512(64, StreamEndianness::Reorder)?;
```

Reply:
Agreed. I need to be better about using them more liberally.